### PR TITLE
Add `/signin/token` to create a verify token without the client flow

### PR DESCRIPTION
**Pull request by Tyrrrz**
JIRA: https://bitwarden.atlassian.net/browse/PAS-224?atlOrigin=eyJpIjoiOTczMDZjN2EwZmIyNDM2NDgyMGE5YzYwNWQzYzUwNmEiLCJwIjoiaiJ9

Request:

```
POST /signin/token

{
  "userId": "user123"
}
```

Response:

```
{
  "token": "verify_asdkasjkdlajscldjaslkdjckasljdc"
}
```

### CODE REVIEW

**Comment by Tyrrrz** [comment]
This is super raw and MVP (would need to add more stuff for magic link support) but works for testing purposes:

image

**Comment by Tyrrrz** [comment]
@abergs @jrmccannon if we can merge it before Monday (ideally before tomorrow), I'll be able to show more interesting tests on the demo :)

**Comment by Tyrrrz** [comment]
Something to consider: what value of `type` should we use for this token?

Currently it's `manual_signin`. Potential ideas: `passkey_manual`, `passkey_magic`, `magic`, `virtual_signin`.

File: src/Service/Fido2ServiceEndpoints.cs [hunk 1]
```diff
@@ -258,6 +252,27 @@ public async Task<TokenResponse> RegisterComplete(RegistrationCompleteDTO reques
         return new TokenResponse(token);
     }
 
+    public Task<string> CreateSigninToken(string userId)
+    {
+        ValidateUserId(userId);
```
Comment (jrmccannon) [inline src/Service/Fido2ServiceEndpoints.cs:257]: Not something that needs to be done now, but do we need to check to see if there's an existing credential? And then invalidate said credential?  That would be something to consider.

**Review by jrmccannon — APPROVED** [review]

**Comment by Tyrrrz** [comment]
I'll merge it for now in this shape since it's only needed for demo and will still require a lot of tweaks for magic links.

The corresponding SDK part can wait.

**Pull request merged**